languages.toml

```
@@ -1448,3 +1448,17 @@ indent = { tab-width = 2, unit = "\t" }
 [[grammar]]
 name = "openscad"
 source = { git = "https://github.com/bollian/tree-sitter-openscad", rev = "5c3ce93df0ac1da7197cf6ae125aade26d6b8972" }
+
+[[language]]
+name = "prisma"
+scope = "source.prisma"
+injection-regex = "prisma"
+file-types = ["prisma"]
+roots = [".git", "package.json"]
```

```suggestion
roots = ["package.json"]
```

we check for `.git` as a default for all languages so no need to add it here: https://github.com/helix-editor/helix/blob/4a27e2d93846f3b66ab0dc225e65d2fb869463f3/helix-loader/src/lib.rs#L87

---

fixed